### PR TITLE
Issue #3218: Replace terms in WritingJavadocChecks wiki-page by terms from HTML specification

### PR DESCRIPTION
**Pull request by subkrish**
Issue #3218 

Replaced the tags as specified in the issue.

### CODE REVIEW

**Comment by subkrish** [comment]
There seems to be only one instance of the singleton tag and none of the beginning tag or the closing tag. Accordingly the change of singleton tag to empty tag in line 318 has been made.

File: src/xdocs/writingjavadocchecks.xml.vm [hunk 1]
```diff
@@ -315,7 +315,7 @@ JAVADOC -> <audio><source src="horse.ogg" type="audio/ogg"/></audio><EOF> [0:0]
 |       |   `--CLOSE -> > [0:6]
 |       |--HTML_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
 |       |   `--SINGLETON_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
-|       |       `--SINGLETON_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
+|       |       `--EMPTY_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
```
Comment (rnveach) [inline src/xdocs/writingjavadocchecks.xml.vm]: As I understand it, this issue is about replacing written paragraph text inside this file, not about changing the printed tree structure.
This tree item is a `SINGLETON_TAG`, so the change is incorrect.

Comment (romani) [inline src/xdocs/writingjavadocchecks.xml.vm]: I created separate issue on this - #4448 .
Should be done when we come to checkstyle8 version

**Comment by romani** [comment]
@rnveach  is right.

AST updates will be done at #4448 (just created issue) , as it is bracking compatibility change

**Pull request closed without merging**